localisation/000_wc_traits.csv

```diff
@@ -371,254 +371,254 @@ mindless_worgen;Mindless;;Hirnlos;;;;;;;;;;;x
+kind_desc;This character is kind and full of empathy. Kind characters may be popular with vassals, but this trait makes them poor spymasters.;;Dieser Charakter ist freundlich und voller Einfühlungsvermögen. Freundliche Charaktere mögen bei Vasallen beliebt sein, aber dieser Charakterzug macht sie zu armseligen Spionen.;;;;;;;;;;;x
+wroth_desc;This character certainly has a hot temper. Sometimes it can fall on others.;;Dieser Charakter hat ganz gewiss ein heißes Temperament. Manchmal kann es sich auf andere auswirken.;;;;;;;;;;;x
 patient_desc;This character is ready to wait as long as necessary. Patient characters are just a little bit better at almost everything.;;Dieser Charakter ist bereit, so lange zu warten wie erforderlich. Geduldige Charaktere sind in fast allem ein wenig besser.;;;;;;;;;;;x
 proud_desc;For some reason this character is proud or even selfish. Proud characters tend to be ambitious and work hard to gain prestige.;;Aus irgendeinem Grund ist dieser Charakter stolz oder sogar egoistisch. Stolze Charaktere neigen dazu, ehrgeizig zu sein und hart zu arbeiten, um Prestige zu erlangen.;;;;;;;;;;;x
-humble_desc;This character is humble and [From.GetHerHis] trait is expressed in moderation in [From.GetHerHis] demands, indifference to wealth and luxury, as well as [From.GetHerHis] determination to show [From.GetHerHis]self. Humble people tend to dislike proud ones.;;Dieser Charakter ist bescheiden und [From.GetHerHis] Charakterzug dr�ckt sich in [From.GetHerHis]en Forderungen aus, in Gleichg�ltigkeit gegen�ber Reichtum und Luxus sowie in [From.GetHerHis]er Entschlossenheit, sich selbst zu zeigen. Bescheidene Leute neigen dazu, Stolze nicht zu m�gen.;;;;;;;;;;;x
-pneumonic_desc;This character suffers from pneumonia � a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentz�ndung, einer oftmals t�dlichen Infektion.;;;;;;;;;;;x
-lefthanded_desc;This character is left-handed. People who are left-handed tend to have a considerable advantage in combat.;;Dieser Charakter ist Linksh�nder. Personen, die Linksh�nder sind, haben im Kampf oft einen erheblichen Vorteil.;;;;;;;;;;;x
+humble_desc;This character is humble and [From.GetHerHis] trait is expressed in moderation in [From.GetHerHis] demands, indifference to wealth and luxury, as well as [From.GetHerHis] determination to show [From.GetHerHis]self. Humble people tend to dislike proud ones.;;Dieser Charakter ist bescheiden und [From.GetHerHis] Charakterzug drückt sich in [From.GetHerHis]en Forderungen aus, in Gleichgültigkeit gegenüber Reichtum und Luxus sowie in [From.GetHerHis]er Entschlossenheit, sich selbst zu zeigen. Bescheidene Leute neigen dazu, Stolze nicht zu mögen.;;;;;;;;;;;x
+pneumonic_desc;This character suffers from pneumonia  a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentzündung, einer oftmals tödlichen Infektion.;;;;;;;;;;;x
```

Comment:
Incorrect symbol
```suggestion
pneumonic_desc;This character suffers from pneumonia - a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentzündung, einer oftmals tödlichen Infektion.;;;;;;;;;;;x
```

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/80320676-7fa3e580-8820-11ea-8a86-c4a540e1ebec.png)

</details>

Reply:
Huh, how did that happen?